src/sentry/static/sentry/app/views/organizationEventsV2/transactionView/dragManager.tsx

```diff
@@ -72,8 +72,11 @@ class DragManager extends React.Component<DragManagerProps, DragManagerState> {
     // prevent the user from selecting things outside the minimap when dragging
     // the mouse cursor outside the minimap
 
-    this.previousUserSelect = document.body.style.userSelect;
-    document.body.style.userSelect = 'none';
+    this.previousUserSelect = setBodyUserSelect({
+      userSelect: 'none',
+      MozUserSelect: 'none',
```

Comment:
Why is `M` capitalized for Moz but not ms?

Reply:
@markstory Good question. I had to look this up to double check.

I got this from https://stackoverflow.com/a/31998812

> Vendor prefixes other than `ms` should begin with a capital letter. 

from: https://reactjs.org/docs/dom-elements.html#style